modules/EmailMan/EmailManDelivery.php

```diff
@@ -248,11 +252,35 @@
             $mail->oe->mail_smtpssl = $outboundEmailAccount->mail_smtpssl;
         }
 
-        if (!$emailman->sendEmail($mail, $massemailer_email_copy, $test)) {
-            $GLOBALS['log']->fatal("Email delivery FAILURE:" . print_r($row, true));
+        if((empty($row['related_confirm_opt_in']) || $row['related_confirm_opt_in'] == '0')) {
+            if (!$emailman->sendEmail($mail, $massemailer_email_copy, $test)) {
+                $GLOBALS['log']->fatal("Email delivery FAILURE:" . print_r($row, true));
+            } else {
+                $GLOBALS['log']->debug("Email delivery SUCCESS:" . print_r($row, true));
+            }
         } else {
-            $GLOBALS['log']->debug("Email delivery SUCCESS:" . print_r($row, true));
+            if($confirmOptInEnabled) {
```

Comment:
is the if a automated formatting issue? should it be elseif?

Reply:
reformatted